packages/lesswrong/components/form-components/TagMultiselect.tsx

```diff
@@ -47,10 +47,11 @@ const TagMultiselect = ({ value, path, classes, label, placeholder, hidePostCoun
 
   const [focused, setFocused] = useState(startWithBorder)
 
-  const addTag = (id: string) => {
-    if (!value.includes(id)) {
-      value.push(id)
-      updateCurrentValues({ [path]: value })
+  const addTag = (id: string, tag: AlgoliaTag | null) => {
+    const ids = [...(tag?.parentTagId ? [tag.parentTagId] : []), id].filter(id => !value.includes(id))
```

Comment:
This will make it add the parent tag anywhere else TagMultiselect is used (e.g. the search page). This may be kind of annoying but I think it's the behaviour we want in most cases